### ISSUE TITLE
Transactions Report - Add missing columns to the exported CSV

### ISSUE THREAD

**Issue by anu-rock** [Automattic/woocommerce-payments#7591]
### Description

Our current CSV export is missing some data points about transactions that would be useful to merchants, esp. medium to large ones, data that are otherwise available on the Transactions list page.

Screenshot of a sample report (transaction ids hidden):

<img width="1264" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/1288616/dcdc1ebd-b61e-4c2d-a13a-58a1d4fe35fc">

### Acceptance criteria

Add new columns to the exported CSV to include the following details about each transaction:

* Currency - 2 columns (useful to spot transactions involving foreign exchange): Customer currency, Deposit currency. Fee (total) will be in the Deposit currency. 
* Change column name from `Amount` to `Amount in Deposit Currency` 
* Payment Method Id (pm_xxx)
* Change column name from `Source` to `Payment Method` to align with the id field
* Clearly indicate the timezone for all timestamps as being UTC
* Deposit Id (i.d. of the deposit batch received from Stripe, generally starts with po_xxxx. This is a useful id for reconciling transactions to deposit batch) 

[Sample report](https://docs.google.com/spreadsheets/d/1WdWxNAAblxqrU9gNjGKm0x-VaWQFMNTJmvPbsrRGLbo/edit?usp=sharing) with updated columns highlighted in yellow. 

```[tasklist]
### Tasks
- [ ] Update Transaction Report API with the field updates as well to maintain parity
- [ ] Update Transaction Report UI with the field updates as well to maintain parity
```

**Comment by souravdebnath1986** [comment]
@anu-rock I have updated the columns slightly. 

Left out the following fields - 
- For `Channel (online or in-person)` - I think this is a part of the csv export and wcadmin. I have also added a task to update the [Transaction Report API ](https://github.com/Automattic/woocommerce-payments/pull/6973)fields as well. 
- Device Type(iOS or Android or card reader model) - We might want to add this field when we have information about Channel = online and device type i.e. whether the transaction was made using desktop web or mobile web
- Device Id (is it possible to get this field, this is a useful field for risk checks)
- Ip Address (I think we collect this)

**Comment by anu-rock** [comment]
Created https://github.com/Automattic/woocommerce-payments/issues/7675 to track the Device Type column separately.
